src/ModelCreator/Downloader/OctokitGraphQLDownloadHelper.cs

```diff
@@ -216,7 +216,7 @@ static OctokitGraphQLDownloadHelper()
                 Items = connection.Nodes.Select(issue => new
                 {
                     Number = issue.Number,
-                    AuthorLogin = issue.Author.Login,
+                    AuthorLogin = issue.Author == null ? null : issue.Author.Login,
```

Comment:
Note we can't use null conditional operator here because it is inside an expression.